### PR TITLE
docs(plugin-ner): Plugin NER Documentation #BOT-322

### PR DESCRIPTION
**Pull request by elozano98**
## Description
Documentation of `plugin-ner`.

### CODE REVIEW

**Review by annehubtype — COMMENTED** [review]
LGTM

**Review by annehubtype — APPROVED** [review]

File: packages/botonic-plugin-ner/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+```bash
+npm install @botonic/plugin-ner
+```
+
+> **Note**: Windows users should first use the command
+> `npm install --global --production windows-build-tools --vs2015`
+
+### Requiring the plugin
+
+The plugin must be required in `src/plugins.js` and, the locales of the trained models must be defined in their options.
```
Comment (annehubtype) [inline packages/botonic-plugin-ner/README.md:24]: and the locales

File: packages/botonic-plugin-ner/README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    options: {
+      locales: ['en', 'es'],
+    },
+  },
+]
+```
+
+## Use
+
+You can now use the recognized entities added by this plugin inside actions to create better conversational flows:
```
Comment (annehubtype) [inline packages/botonic-plugin-ner/README.md:42]: The plugin has just added the recognized entities. Now you can use them within actions to create better conversational flows:

File: packages/botonic-plugin-ner/README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
+# Botonic Plugin NER
+
+## What does this plugin do?
+
+Botonic Plugin NER uses the trained models defined by the user to recognize named entities inside the input text.
+
+After detecting the text locale, and using the corresponding model to recognize the entities, extra information about the recognized entities is added to the input message.
```
Comment (annehubtype) [inline packages/botonic-plugin-ner/README.md:7]: First, the text locale is detected. Then, by using the relevant model, extra information about the recognized entities is added to the input message.

**Review by annehubtype — APPROVED** [review]

**Review by dpinol — APPROVED** [review]

**Review by vanbasten17 — APPROVED** [review]

**Pull request merged**